Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev.py

```diff
@@ -0,0 +1,149 @@
+        else:
+            if "Key" not in kv_pair or "Value" not in kv_pair:
+                demisto.info(f"Ignoring item because it lacks the keys 'Key' and/or 'Value': {sorted(kv_pair.keys())}")
+            else:
+                key = _canonicalize_string(kv_pair.get("Key", ""))
+                value = _canonicalize_string(kv_pair.get("Value", ""))
+
+                if ("env" in key) or (key in ("stage", "function", "lifecycle", "usage", "tier")):
+                    if (("dev" in value and "devops" not in value)
+                            or ("uat" in value and "prod" not in value)
+                            or any([m == value for m in EXACT_DEV_MATCH])
+                            or any([m in value for m in PARTIAL_DEV_MATCH])):
+                        has_dev_indicator = True
+                    elif (any([m == value for m in EXACT_PROD_MATCH])
+                          or any([m in value for m in PARTIAL_PROD_MATCH])):
+                        has_prod_indicator = True
+
+    if has_dev_indicator and not has_prod_indicator:
+        return True
+    else:
+        return False
+
+
+def is_dev_according_to_classifications(classifications: List[str]) -> bool:
+    """
+    Returns whether any of the classification strings indicate that this service is
+    a development environment. The Xpanse ASM classification strings are a defined
+    vocabulary of facts that are derivable from the publicly observed service without
+    joining on any additional data.
+
+    Args:
+        classifications (List[str]): list of Xpanse ASM classification terms
+            (a defined vocabulary)
+            Example value for `classifications`:
+                ["RdpServer", "SelfSignedCertificate"]
+
+    Returns:
+        bool: whether there is an indication within `classifications` that
+            the described system is used for development
+    """
+    DEV_ENV_CLASSIFICATION = "DevelopmentEnvironment"
+    is_dev = (DEV_ENV_CLASSIFICATION in classifications)
+    return is_dev
+
+
+""" MAIN FUNCTION """
+
+
+def main():
+    """
+    Identifies whether the service is a "development" server. Development servers
+    have no external users and run no production workflows. These servers might be
+    named "dev", but they might also be named "qa", "pre-production", "user
+    acceptance testing", or use other non-production terms. This automation uses
+    both public data visible to anyone (`active_classifications` as derived by
+    Xpanse ASM) as well as checking internal data for AI-learned indicators of
+    development systems (`asm_tags` as derived from integrations with non-public
+    systems).
+
+    Args:
+        asm_tags (List[Dict[str, Any]]): list of key-value dictionaries;
+            each dictionary within the list must contain the keys "Key" and "Value";
+            the values are arbitrary
+            Example value for `observed_key_value_pairs`:
+                [{"Key": "env", "Source": "AWS", "Value": "dev"},
```

Comment:
[{"Key": "env", "Source": "AWS", "Value": "dev"}
```suggestion
                [{"Key": "env", "Value": "dev"},
```

you dont check the `Source` key at all, please remove from the examples